src/logger.middleware.ts

```diff
@@ -0,0 +1,11 @@
+import { Injectable, NestMiddleware } from '@nestjs/common';
+import { Request, Response, NextFunction } from 'express';
+
+@Injectable()
+export class LoggerMiddleware implements NestMiddleware {
+    use(req: Request, res: Response, next: NextFunction) {
+        const clientIp = req.ip || req.socket.remoteAddress;
+        console.log(`Request from IP: ${clientIp}`);
+        next();
+    }
+}
```

Comment:
Better name it to LogIpMiddleware because we only log ip here
and move it please to src/middlewares/